packages/deployment-service-ui/src/components/pages/releases.tsx

```diff
@@ -14,6 +15,9 @@ export default () => {
   const [deployLoading, setDeployLoading] = useState<string | undefined>()
 
   const { projectName } = useParams<{ projectName: string }>()
+  // const snack = useSnack()
```

Comment:
can u delete the commented stuff?

Reply:
Suspect this could be commented out because you need to wrap the app in the <SnackProvider>